### ISSUE TITLE
Building Relic with SGX Enclave-based Application

### ISSUE THREAD

**Issue by mayank0403** [relic-toolkit/relic#96]
I was trying to integrate Relic with my SGX enclave application code. In particular, I would like to build Relic with SGX enclave flags. This will require removing a lot of functionalities like printing, system call, streams, etc. from Relic.
If my application, say uses only the eb functionalities, can I export just those functionalities from Relic and then include them in my enclave application, therefore building the application with only the required functionalities?
Basically, how would one go about building just some functionalities of Relic along with a regular cpp application?

**Comment by dfaranha** [comment]
I believe so, but have no previous experience with this. Perhaps you should try asking help in the mailing list instead.

**Comment by mayank0403** [comment]
Thanks. I will give it an effort and if I fail, I will ask for help in the mailing list.
